node/grpc/utils.go

```diff
@@ -40,15 +43,15 @@ func GetBlobMessages(in *pb.StoreChunksRequest) ([]*core.BlobMessage, error) {
 		}
 
 		bundles := make(map[core.QuorumID]core.Bundle, len(blob.GetBundles()))
-		for i, chunks := range blob.GetBundles() {
-			quorumID := blob.GetHeader().GetQuorumHeaders()[i].QuorumId
+		for j, chunks := range blob.GetBundles() {
```

Comment:
On line 41, Since GetHeader() can be nil, GetQuorumHeaders() will evaluate to invalid memory. Similarly for L44

Reply:
I added a node request validation here: https://github.com/Layr-Labs/eigenda/pull/378, to make sure header is non-null

Reply:
Let's add a comment in L33 that this method assumes the inputs are valid and point to the validation method